src/Solnet.Rpc/Models/Logs.cs

```diff
@@ -34,4 +36,38 @@ public class LogInfo : Log
         /// </summary>
         public string Signature { get; set; }
     }
+
+    public class InstructionError
+    {
+        public int ErrorCode { get; set; }
+        
+        public KeyValuePair<string, int> CustomError { get; set; }
+    }
+    
+    public class InstructionErrors
+    {
+        [JsonPropertyName("InstructionError")]
+        [JsonConverter(typeof(InstructionErrorJsonConverter))]
+        public InstructionError[] InstructionError { get; set; }
```

Comment:
I was thinking more along the lines of a list<object> or object[]. Are we sure that there are no more types of errors other than these two fields within InstructionError ?

Reply:
No but it can be refactored later, so far I've only seen these types when attempting transactions with possible errors

Reply:
I guess we could create an issue for a later milestone (0.2 or 1.0?) to review the rust code and try to find out what we can actually find out in these errors?!